src/lib/browse/colors.ts

```diff
@@ -10,6 +10,8 @@ export const colors = {
   combined_authorities: "cyan",
   local_authority_districts: "orange",
   local_planning_authorities: "red",
+  bus_route_with_lane: "#9362BA",
```

Comment:
Just kind of picked a color and then found a darker shade of it. Suggestions very welcome

Reply:
I like it